lib/js/components/ProgressBar/ProgressBar.vue

```diff
@@ -116,29 +116,29 @@ $progress-bar-layers: (
 	@each $color-name, $color-map in $progress-bar-layers {
 		&.-#{$color-name} {
 			#{$self}__result {
-				background: map-get($color-map, 'default');
+				background: map-get($color-map, 'default-color-scheme-first-layer');
 
 				&.-secondary {
-					background: map-get($color-map, 'ghost');
+					background: map-get($color-map, 'default-color-scheme-second-layer');
 				}
 			}
 
 			&.-schemeMedium {
 				#{$self}__result {
-					background: map-get($color-map, 'medium');
+					background: map-get($color-map, 'medium-color-scheme-first-layer');
 
 					&.-secondary {
-						background: map-get($color-map, 'weak');
+						background: map-get($color-map, 'medium-color-scheme-second-layer');
 					}
 				}
 			}
 
 			&.-schemeMediumNeutral {
 				#{$self}__result {
-					background: map-get($color-map, 'medium');
+					background: map-get($color-map, 'medium-color-scheme-first-layer');
```

Comment:
I'm not fan of reusing color from other color scheme, but don't have strong arguments to convince you not to do that